modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -247,6 +276,9 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPacketsForChannel() {
 				suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), identifiedPacketFees.PacketId, types.NewPacketFees(identifiedPacketFees.PacketFees))
 			}
 
+			path := ibctesting.NewTransferPathWithFeeEnabled(suite.chainA, suite.chainB)
+			path.Setup()
```

Comment:
I tried but it is not working on those tests, since the `req` that we build uses values (`channelID`,`portID`) that, based on my understanding, are set when calling `setup()`). So we need them to be set before (or simultaneously) we call the `tc.malleate()` function

Reply:
ah I see, in that case I think it's fine to leave as is :+1: